src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -223,9 +213,178 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
-    <!-- TODO -->
+    <Compile Include="System\Net\Sockets\_AcceptOverlappedAsyncResult.Windows.cs" />
+    <Compile Include="System\Net\Sockets\_BaseOverlappedAsyncResult.Windows.cs" />
+    <Compile Include="System\Net\Sockets\_ConnectOverlappedAsyncResult.Windows.cs" />
+    <Compile Include="System\Net\Sockets\_OverlappedAsyncResult.Windows.cs" />
+    <Compile Include="System\Net\Sockets\Socket.Windows.cs" />
+    <Compile Include="System\Net\Sockets\SocketAsyncEventArgs.Windows.cs" />
+    <Compile Include="System\Net\Sockets\SocketPal.Windows.cs" />
+
+    <Compile Include="$(CommonPath)\System\Net\SafeCloseSocket.Windows.cs">
+      <Link>Common\System\Net\SafeCloseSocket.Windows.cs</Link>
+    </Compile>    
+    <Compile Include="$(CommonPath)\System\Net\SocketAddressPal.Windows.cs">
+      <Link>Common\System\Net\SocketAddressPal.Windows.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\System\Net\SocketProtocolSupportPal.Windows.cs">
+      <Link>Common\System\Net\SocketProtocolSupportPal.Windows</Link>
+    </Compile>
+
+    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
+      <Link>Interop\Windows\Interop.Libraries.cs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="System\Net\Sockets\_AcceptOverlappedAsyncResult.Unix.cs" />
+    <Compile Include="System\Net\Sockets\_BaseOverlappedAsyncResult.Unix.cs" />
+    <Compile Include="System\Net\Sockets\_ConnectOverlappedAsyncResult.Unix.cs" />
+    <Compile Include="System\Net\Sockets\_OverlappedAsyncResult.Unix.cs" />
+    <Compile Include="System\Net\Sockets\BufferList.cs" />
+    <Compile Include="System\Net\Sockets\Socket.Unix.cs" />
+    <Compile Include="System\Net\Sockets\SocketAsyncEventArgs.Unix.cs" />
+    <Compile Include="System\Net\Sockets\SocketPal.Unix.cs" />
+
+    <!-- TODO: remove this -->
+    <Compile Include="RefactoringShims\InteropShim.cs">
```

Comment:
The references come from the Windows interop files that are still included when building for *nix.